edx_name_affirmation/api.py

```diff
@@ -194,23 +146,29 @@ def update_verified_name_status(
         * status (Verified Name Status)
         * verification_attempt_id (int)
         * proctored_exam_attempt_id (int)
+        * platform_verification_attempt_id (int)
```

Comment:
Could we please update the comments here? I feel like with the platform_verification_attempt_id, the comments, should be more specific about how it should work with verification_attempt_id